tests/ui/async-await/debug-ice-attempted-to-add-with-overflow.rs

```diff
@@ -0,0 +1,10 @@
+// check-fail
+// edition:2021
+
+// test for issue-114912 - debug ice: attempted to add with overflow
+
+async fn main() {
+    //~^ ERROR E0752
+    [0usize; 0xffff_ffff_ffff_ffff].await;
+    //~^ ERROR E0277
```

Comment:
```suggestion
    //~^ error[E0277]: `[usize; usize::MAX]` is not a future
```

As adding the error code is super helpful usually I do not know what it means,
and I preferer to read the error. This also prevents the message content regression.